apps/launchpad/app/routes/_app+/quests+/questions+/question+/$id.tsx

```diff
@@ -32,42 +32,94 @@ import {
 import { QUESTIONS_METADATA } from '@lib/utils/constants'
 import logger from '@lib/utils/logger'
 import { usePrivy } from '@privy-io/react-auth'
+import type { LoaderFunctionArgs, MetaFunction } from '@remix-run/node'
 import { useLoaderData, useParams } from '@remix-run/react'
 // import { requireUser } from '@server/auth'
 import { dehydrate, QueryClient } from '@tanstack/react-query'
 import { useAtom } from 'jotai'
 import { formatUnits } from 'viem'
 
-export async function loader() {
+export async function loader({ request }: LoaderFunctionArgs) {
   const queryClient = new QueryClient()
-  // const user = await requireUser(request)
-  // const wallet = user?.wallet?.address
-
-  await queryClient.prefetchQuery({
-    queryKey: ['get-list-details', { predicateId: 3, objectId: 620 }],
-    queryFn: () =>
-      fetcher<GetListDetailsQuery, GetListDetailsQueryVariables>(
-        GetListDetailsDocument,
-        {
-          tagPredicateId: 3,
-          globalWhere: {
-            predicate_id: {
-              _eq: 3,
-            },
-            object_id: {
-              _eq: 620,
-            },
-          },
-        },
-      ),
-  })
+
+  const variables = {
+    tagPredicateId: 3,
```

Comment:
In my recent PR I started moving away from these in favor of using `getSpecialPredicate` -- may be worth fixing now